src/Components/ReviewForm.js

```diff
@@ -1,29 +1,34 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-
-class ReviewForm extends Component {
-    constructor(props) {
-        super(props);
-
-    }
-
-
-    componentDidMount() {
-
-    }
-
-
-    render() {
-        return (
-            <div>
-
-            </div>
-        );
-    }
-}
-
-ReviewForm.propTypes = {
-
+import React from 'react';
+
+const ReviewForm = () => {
+    return (
+        <div>
+            <header>
+                <h2 className="form-heading">Enter new client information here to keep yourself and your community in the know about the current ClientTell out there. </h2>
+            </header>
+             <div className="form-container">
+                 <p>Enter either or both of the following: </p>
+            <input
+                id="phoneNumberInput"
+                type="text"
+                name="phoneNumber"
+                placeholder="Phone Number"
+                // value=""
```

Comment:
this is where useState will come in... onChange it will setUser/setPhoneNumber... and we will need to pass that into the ReviewFormDetails so that they are submitted with all of the details when we make the POST